core-plugins/src/e2e-test/java/io/cdap/plugin/joiner/locators/JoinerLocators.java

```diff
@@ -28,7 +28,17 @@ public class JoinerLocators {
 
   public static WebElement fieldAliasCheckBox(String pluginName, String field) {
     String xpath = "//*[@data-cy='" + pluginName + "-stage-expansion-panel']" + "//*[@data-cy='" + field +
-      "-field-selector-name']/..//*[@type='checkbox']";
+      "-field-selector-name']/..//*[@data-cy='"+ field +"-field-selector-checkbox']";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  public static WebElement requiredInputCheckboxFirstFile(String pluginName){
```

Comment:
Update: pluginName to input. 
Remove the second locator and use the index as a parameter.